### DISCUSSION TITLE
Need Help with How to Access Pywebview.Window

### DISCUSSION THREAD

**Discussion by KiddoV** [r0x0r/pywebview#776] [Q&A] [answered]
Can anyone answer this question for me? I have been struggling with this for days. Thank you!

https://stackoverflow.com/questions/69020223/how-to-access-pywebview-window-object-from-another-multiprocessing-process

**Comment by r0x0r** [comment, marked as answer]
Look into Exchanging objects between processes in https://docs.python.org/3/library/multiprocessing.html
